lib/routes/account.js

```diff
@@ -35,6 +35,9 @@ module.exports = function (
   ) {
 
   var verificationReminder = require('../verification-reminders')(log, db)
+  var getGeoData = require('../geodb')(log, {
+    shouldReturnGeodata: true
```

Comment:
I mean the "kill switch" should probably be set in `config/index.js` and you don't need to pass any options here. 
Just use the setting from the config object given to you here: https://github.com/mozilla/fxa-auth-server/pull/1334/files#diff-4d35aa03e69653f3c13a956d79d3b242R4